installer/install_delfin.py

```diff
@@ -139,11 +139,6 @@ def main():
                                  'delfin', 'delfin.conf')
     copy_files(conf_file_src, conf_file)
 
-    # Copy the scheduler_config.json file
```

Comment:
Need to remove installer reference , opened https://github.com/sodafoundation/delfin/issues/507 in delfin and https://github.com/sodafoundation/installer/issues/426 in installer

Reply:
Sure, The current code removal is part of scheduler conf related removal.